field_records/ubna_2023.csv

```diff
@@ -73,10 +73,22 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
 2023-08-09T14:24:00 , 2023-08-15T09:54:00 , E , 006 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 192000 , Medium , None , None , 4.135 , 3.405 , CT , AK , CT , recover-20230815 , Ikea Batteries Nighttime 3:00-13:30 UTC;
 2023-08-09T14:24:00 , 2023-08-15T09:54:00 , F , 007 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 48000 , Medium , None , None , 4.135 , 3.671 , CT , AK , CT , recover-20230815 , Ikea Batteries Daytime 0:00-24:00 UTC;
 2023-08-09T14:47:00 , 2023-08-15T09:41:00 , C , 011 , E18 Bridge , 47° 39' 21.432'' N , 122° 17' 48.612'' W , 192000 , Medium , None , None , 4.068 , 3.450 , CT , AK , CT , recover-20230815 , Panasonic Batteries Nighttime 3:00-13:30 UTC; SD card 'UBNA_011' from recover-20230809 was used here because we did not bring 7 SD cards and 'UBNA_011' had popped out so it was empty;
-2023-08-15T10:20:00 , (DATE-TIME) , H , 005 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 48000 , Medium , None , None , 4.183 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
-2023-08-15T10:20:00 , (DATE-TIME) , B , 015 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 192000 , Medium , None , None , 4.173 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:00-13:30 UTC;
-2023-08-15T10:09:00 , (DATE-TIME) , G , 001 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 192000 , Medium , None , None , 4.096 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC;
-2023-08-15T10:09:00 , (DATE-TIME) , D , 016 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 48000 , Medium , None , None , 4.115 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Panasonic Batteries Daytime 0:00-24:00 UTC;
-2023-08-15T09:57:00 , (DATE-TIME) , E , 010 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 192000 , Medium , None , None , 4.170 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC;
-2023-08-15T09:57:00 , (DATE-TIME) , F , 002 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 48000 , Medium , None , None , 4.186 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC;
-2023-08-15T09:43:00 , (DATE-TIME) , C , 003 , E18 Bridge , 47° 39' 21.432'' N , 122° 17' 48.612'' W , 192000 , Medium , None , None , 4.136 , (VOLTAGE) , MB , CT , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC;
+2023-08-15T10:20:00 , (DATE-TIME) , H , 005 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 48000 , Medium , None , None , 4.183 , (VOLTAGE) , MB , CT , CT , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC; Missing;
```

Comment:
Could you elaborate on "Missing" and say "Audiomoth has gone missing in the field" or something?

Reply:
I will not forget this (again)!